hack/update-all.sh

```diff
@@ -24,3 +24,4 @@ for f in ${HACK_DIR}/update-*.sh; do
   fi
   $f
 done
+go mod tidy
```

Comment:
I wouldn't want tidy to be run by these scripts..

Could certainly add to github actions ci to check if it's tidy, but automatically modifying go.mod isn't ideal for me.